crates/blockchain/Cargo.toml

```diff
@@ -38,3 +38,4 @@ metrics = ["ethrex-metrics/transactions"]
 
 [lints.clippy]
 unwrap_used = "deny"
+result_large_err = "allow"
```

Comment:
I think we don't want to allow this. Having large errors can lead to performance issues in frequently called functions, so a local allow makes more sense.